### ISSUE TITLE
Data terminal Broken

### ISSUE THREAD

**Issue by Silvieson** [NeoArmageddon/co10_Escape#94]
Been playing co10 escape cup and regardless of the map I do not get any interaction options on the data terminal

mods I use are: ACE, CBA and the CUP mods.

I see the data terminal but I get no option to interact with the arma scroll function nor either ACE interaction menu.

![AD9DB043-C1A0-4A20-86C3-3EAB7920FC91](https://user-images.githubusercontent.com/77470471/104666233-7bf61d00-5727-11eb-955e-06f3978a2d4f.jpeg)

**Comment by NeoArmageddon** [comment]
Release version 1.10? Just played and worked flawless for me and my squad.
Is the boxs light in the screenshot green or white? If it is green, the script adding the action was definitly called correctly so no action menu entry is probably the fault of an addon. If the box light is white means there is something seriously broken with your server as no script was called. Check your RPT.

**Comment by Dystopian** [comment]
@Silvieson also you have to scroll down/up to get action to see it, it's not shown just on cursor hover (at least for me)

**Comment by Silvieson** [comment]
> Release version 1.10? Just played and worked flawless for me and my squad.
> Is the boxs light in the screenshot green or white? If it is green, the script adding the action was definitly called correctly so no action menu entry is probably the fault of an addon. If the box light is white means there is something seriously broken with your server as no script was called. Check your RPT.

Definitely green mate. Will remove all but CUP and see how it goes

> @Silvieson also you have to scroll down/up to get action to see it, it's not shown just on cursor hover (at least for me)

doesn’t work with scroll either for me

**Comment by NeoArmageddon** [comment]
Also please post your .RPT error log from a session the terminal was not hackable.

**Comment by Silvieson** [comment]
[arma3server_x64_2021-01-16_21-11-11.rpt.txt](https://github.com/NeoArmageddon/co10_Escape/files/5824289/arma3server_x64_2021-01-16_21-11-11.rpt.txt)

**Comment by Silvieson** [comment]
> 
> 
> Also please post your .RPT error log from a session the terminal was not hackable.

I completely started from scratch tonight server wise and tried utes. same issue box has green lights but no option in the scroll menu to hack the box. ive attached my RPT
[arma3server_x64_2021-01-16_21-11-11.rpt.txt](https://github.com/NeoArmageddon/co10_Escape/files/5824293/arma3server_x64_2021-01-16_21-11-11.rpt.txt)

**Comment by NeoArmageddon** [comment]
Can you also provide the following files:
D:\SteamLibrary\steamapps\common\Arma 3 Server\TADST\default\TADST_basic.cfg
D:\SteamLibrary\steamapps\common\Arma 3 Server\TADST\default\TADST_config.cfg
Or clients computer rpt from the same session (Username/AppData/Local/Arma3/*.rpt)

Please disable the co10 Escape addons and try with the mission files instead.

As of now I suspect TADST to be the problem.

**Comment by NeoArmageddon** [comment]
Next time you are testing and there is still no action, you can try to enter this in the debug console and execute locally when looking at the terminal:
`cursortarget addAction ["Hack test", {hint "hacking...";_this call A3E_fnc_Hijack;}];`

**Comment by Silvieson** [comment]
> 
> 
> Can you also provide the following files:
> D:\SteamLibrary\steamapps\common\Arma 3 Server\TADST\default\TADST_basic.cfg
> D:\SteamLibrary\steamapps\common\Arma 3 Server\TADST\default\TADST_config.cfg
> Or clients computer rpt from the same session (Username/AppData/Local/Arma3/*.rpt)
> 
> Please disable the co10 Escape addons and try with the mission files instead.
> 
> As of now I suspect TADST to be the problem.

I Dont believe that it is TADST because ive only just started using it again and the issue existed before i reused that application.

i originally had written my own config file, but decided to go with TADST with my new setup to simply the setup. however i will provide the requested files, i apologize for taking so long ive been busy with work. 

Ive renamed the file types to txt so they allwed me to upload it.

Im planning on playing sometime tonight without ACE so i will report back.

[TADST_basic.txt](https://github.com/NeoArmageddon/co10_Escape/files/5834033/TADST_basic.txt)
[TADST_config.txt](https://github.com/NeoArmageddon/co10_Escape/files/5834034/TADST_config.txt)

[Arma3_x64_2021-01-16_21-12-47.txt](https://github.com/NeoArmageddon/co10_Escape/files/5834055/Arma3_x64_2021-01-16_21-12-47.txt)

**Comment by NeoArmageddon** [comment]
No (escape) errors in your RPT and all debug messages are shown, so every script ran correctly and the terminal action was added.
Thou there are ACE error messages.
Can you run `[] call A3E_fnc_addUserActions;` the next time you encounter a terminal without action in your console? Of that does not work, there is something seriously wrong with your mod/addon setup.

I suspect  some mod or addon is running [removeAllActions](https://community.bistudio.com/wiki/removeAllActions) on the player and removing all actions.

**Comment by Silvieson** [comment]
so with ACE and its cup comparability mods disabled hack terminal now works..

**Comment by NeoArmageddon** [comment]
Thanks, that is what I suspected. ACE seems to mess with the actions added to the player after the mission start. Nothing I can do here but I will try to come up with a workaround eventually.

**Comment by Dystopian** [comment]
I play with ACE (without medical component) and CUP for some years - no problems at all.

**Comment by NeoArmageddon** [comment]
> I play with ACE (without medical component) and CUP for some years - no problems at all.

Let me rephrase it: The problem is not with the mission but with the version of ACE and CUP compat or any other addon loaded alongside or with a specific subsystem of ACE, like medical. Fixing this is out of the scope of this mission.

**Comment by Silvieson** [comment]
I will keep troubleshooting it and see if there is a particular part of ace that causes it. Maybe it is ACE Medical as it would conflict with your unconscious script perhaps.

**Comment by NeoArmageddon** [comment]
Have you tested just pressing "Spacebar" (or the default action key if changed in your control preset) when looking at the terminal and not having selected another action from the action menu?

While testing in 3Den I didn't got the action displayed but could nevertheless start hacking like described above.
